packages/sit-onyx/src/components/examples/GridPlayground/GridElement/GridElement.vue

```diff
@@ -0,0 +1,74 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+import type { GridElementConfig } from "../EditGridElementDialog/EditGridElementDialog.vue";
+
+const props = defineProps<
+  GridElementConfig & {
+    label: string;
+    mode?: "default" | "outline";
+  }
+>();
+
+const emit = defineEmits<{
+  click: [];
+}>();
+
+const gridClasses = computed(() => {
+  return [
+    `onyx-grid-span-${props.columnCount}`,
+    ...Object.entries(props.breakpoints ?? {}).map(([breakpoint, columns]) => {
+      return `onyx-grid-${breakpoint}-span-${columns}`;
+    }),
+  ];
+});
+</script>
+
+<template>
+  <button
+    class="grid-element"
+    :class="[...gridClasses, props.mode === 'outline' ? 'grid-element--outline' : '']"
+    type="button"
+    :aria-label="props.label"
+    @click="emit('click')"
+  >
+    <slot></slot>
```

Comment:
wouldn't this be a great space to show the current number of columns in here when there is no icon present? (you could clarify with Jannick first and make a follow-up PR if you want :))
Right now you always have to manually compare to the colored line above to verify the current count, and if it is more than one row, maybe hold a ruler 📐 on your screen to track it xD (_or use dev tools, of course :)_)

of course, only if that wouldn't get too complicated / duplicate logic that is currently only covered by CSS

Reply:
Do you mean something like this? 
![image](https://github.com/user-attachments/assets/322d17e0-2811-46d8-bbab-a293eb307558)

Reply:
yes! that would be awesome 😊